stable/odoo/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: postgresql
+  version: 0.2.1
```

Comment:
Change this to point to 0.3.0 which has the fix for imagePullPolicy on the init container.